composer.json

```
@@ -43,6 +43,7 @@
     "symfony/process": "^5.4|^6.4"
   },
   "require-dev": {
+	  "fluidtypo3/vhs": "^6.1",
```

Das müsste 7.0 sein, damit TYPO3 v11 und v12 unterstützt werden.

```suggestion
    "fluidtypo3/vhs": "^7.0",
```